gpao/builder.py

```diff
@@ -3,12 +3,15 @@
 import json
 import requests
 from gpao.project import Project
+from gpao.job import Job
 
 
 def handler(obj):
```

Comment:
je sais pas si ç vaut le coup, mais pour éviter une copie où tu enlèves plein de choses, tu peux transformer ta fonction en quelque chose comme
```suggestion
def handler(obj):
    """ handler"""
    noise_keys = ["internal_id", "reorganized"]
     dictionary = {k: v for k, v in obj.__dict__.items() if v and (k not in noise_keys)}
     
    return dictionary

```

Reply:
resolu